test/ledger/wallet.integration.test.js

```diff
@@ -170,7 +176,7 @@ test('compositing wallet grant information', async (t) => {
   } = await ledgerAgent.get(`/v2/wallet/${paymentId}/grants/ads`).expect(ok)
   const expectedAds = createComposite({
     type: 'ads',
-    amount: (new BigNumber(2)),
+    amount: (new BigNumber(2)).plus(13),
```

Comment:
why do we add 13 here?

Reply:
that's for the bonus. the bonus is `25` and the value in the grant is `38` so 38-25=13

Reply:
so it's as if they earned 25 in the bonus and 13 from doing the work of clicking on ads